### PR TITLE
docs: 1.3.0 release notes

### PR DESCRIPTION
**Pull request by mattklein123**

### CODE REVIEW

**Comment by mattklein123** [comment]
@lyft/network-team and everyone else. Please have a look. There may be a few straggling things that we get in here but this is pretty close.

File: docs/intro/version_history.rst [hunk 1]
```diff
@@ -1,6 +1,74 @@
...
+  </CONTRIBUTING.md#breaking-change-policy>`.
+* Bazel is now the canonical build system (replacing CMake). There have been a huge number of
+  changes to the development/build/test flow. See :repo:`/bazel/README.md` and
+  :repo:`/ci/README.md` for more information.
+* :ref:`Outlier detection <arch_overview_outlier_detection>` has been expanded to include success
+  rate variance, and all parameters are now configurable in both runtime and in the JSON
+  configuration.
+* TCP level :ref:`listener <config_listeners_per_connection_buffer_limit_bytes>` and
+  :ref:`cluster <config_cluster_manager_cluster_per_connection_buffer_limit_bytes>` connections now
+  have configurable buffer limits at which point connection level back pressure is applied. Full end
```
Comment (htuch) [inline docs/intro/version_history.rst]: We technically don't have back pressure yet, we're only using them on the receive side to ensure we start pushing the buffers down the filter chain when we hit the limits.

File: docs/intro/version_history.rst [hunk 2]
```diff
@@ -1,6 +1,74 @@
...
+  desired.
+* HTTP routes can now add request headers on a per route and per virtual host basis via the
+  :ref:`request_headers_to_add <config_http_conn_man_route_table_route_add_req_headers>` option.
+* The :ref:`example configurations <install_ref_configs>` have been refreshed to demonstrate the
+  latest features.
+* :ref:`per_try_timeout_ms <config_http_conn_man_route_table_route_retry>` can now be configured in
+  a route's retry policy in addition to via the :ref:`x-envoy-upstream-rq-per-try-timeout-ms
+  <config_http_filters_router_x-envoy-upstream-rq-per-try-timeout-ms>` HTTP header.
+* :ref:`HTTP virtual host matching <config_http_conn_man_route_table_vhost>` now includes support
+  for wildcard domains.
```
Comment (rshriram) [inline docs/intro/version_history.rst]: I think this is just prefix wildcard or suffix. Not arbitrary wildcard is it?

File: docs/intro/version_history.rst [hunk 3]
```diff
@@ -1,6 +1,74 @@
...
+* The default for tracing random sampling has been changed to 100% and is still configurable in
+  :ref:`runtime <config_http_conn_man_runtime>`.
+* :ref:`HTTP tracing configuration <config_http_conn_man_tracing>` has been extended to allow tags
+  to be populated from arbitrary HTTP headers.
+* The :ref:`HTTP rate limit filter <config_http_filters_rate_limit>` can now be applied to internal,
+  external, or all requests via the `request_type` option.
+* :ref:`Listener binding <config_listeners>` now requires specifying an `address` field. This can be
+  used to bind a listener to both a specific address as well as a port.
```
Comment (rshriram) [inline docs/intro/version_history.rst]: This bit needs to move to top and probably be called out as a breaking change in the config.

**Comment by rshriram** [comment]
And do you expect the zipkin stuff to get in by 1.3 time frame ?

File: docs/intro/version_history.rst (hunk 1 above)
Comment (mattklein123) [inline docs/intro/version_history.rst]: Will update.

File: docs/intro/version_history.rst (hunk 2 above)
Comment (mattklein123) [inline docs/intro/version_history.rst]: Yup will fix.

File: docs/intro/version_history.rst (hunk 3 above)
Comment (mattklein123) [inline docs/intro/version_history.rst]: The breaking change policy starts at 1.3.0, so I'm not going to list out every breaking change. I will clarify the text a bit to make this clear.

**Comment by mattklein123** [comment]
> And do you expect the zipkin stuff to get in by 1.3 time frame ?

I will take a look at the PR tomorrow and try to get a feel for how close it is. Releases are obviously totally arbitrary and most people are running master, so making it by a particular release is not super important IMO.

File: docs/intro/version_history.rst [hunk 4]
```diff
@@ -1,6 +1,74 @@
...
+* A :ref:`route table checking tool <install_tools>` has been added that can be used to test route
+  tables before use.
+* We have added an :ref:`example repo <extending>` that shows how to compile/link a custom filter.
+* Added additional cluster wide information related to outlier detection to the :ref:`/clusters
+  admin endpoint <operations_admin_interface>`.
+* Multiple SANs can now be verified via the :ref:`verify_subject_alt_name
+  <config_listener_ssl_context>` setting. Additionally, URI type SANs can be verified.
+* HTTP filters can now be passed :ref:`opaque configuration
+  <config_http_conn_man_route_table_opaque_config>` specified on a per route basis.
+* By default Envoy now has a built in crash handler that will print a back trace. This behavior can
```
Comment (rshriram) [inline docs/intro/version_history.rst]: Add zipkin tracing support?

Comment (mattklein123) [inline docs/intro/version_history.rst]: Yes will refresh this PR today.

**Comment by mattklein123** [comment]
@rshriram @htuch @lyft/network-team updated, PTAL.

**Pull request merged**